src/DotNet/ImplMap.cs

```diff
@@ -215,6 +215,7 @@ public abstract class ImplMap : IMDTokenProvider {
 		}
 
 		static string GetDllName(string dllName) {
+			dllName = dllName.TrimEnd(' ');
```

Comment:
Your post mentions that all whitespace is ignored. You can pass in `null` here and all whitespace will be trimmed.